docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/terraform-module-scan.adoc

```diff
@@ -0,0 +1,22 @@
+== Terraform Module Scanning
+
+Prisma Cloud supports rendering and scanning both public, private, and locally cached Terraform modules (modules). This capability enables users to analyze misconfigurations that may result from the module definition and the variables defined in the module block.
+
+image::application-security/tf-module-scan.png[]
+
+=== Public and Locally Cached Modules
+
+Automated support for public modules is provided by the platform for all version control system (VCS) repositories that have been onboarded (For information on onboarding repositories refer to xref:../../get-started/connect-code-and-build-providers/code-repositories/code-repositories.adoc[Code Repositories]).
+
+==== Public and Local Module Scanning through Checkov CLI
+
+When using the Checkov CLI (CLI), make sure to include the `--download-external-modules true` flag to enable the automatic download of external modules.
+
+Additionally, the CLI supports scanning the `.terraform` directory when modules have been downloaded locally. Note that nested modules are not yet supported in scans performed in the  `.terraform` directory.
```

Comment:
The last sentence should be a stand-alone Note.

Reply:
Done